vendor/github.com/opencontainers/runc/vendor.conf

```diff
@@ -1,8 +1,9 @@
 # OCI runtime-spec. When updating this, make sure you use a version tag rather
 # than a commit ID so it's much more obvious what version of the spec we are
 # using.
-github.com/opencontainers/runtime-spec 5684b8af48c1ac3b1451fa499724e30e3c20a294
+github.com/opencontainers/runtime-spec 29686dbc5559d93fb1ef402eeda3e35c38d75af4
```

Comment:
do we want the vendored version to match this one?

Reply:
I think it makes sense to use the runtime-spec that matches runc; IMO

Reply:
ok, I'll update this PR, and update that as well, thanks!